SteamKit2/SteamKit2/Types/GameID.cs

```diff
@@ -64,6 +64,36 @@ public GameID( Int32 nAppID )
             : this( ( UInt64 )nAppID )
         {
         }
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GameID"/> class.
+        /// </summary>
+        /// <param name="nAppID">The base app id of the mod.</param>
+        /// <param name="modPath">The game folder name of the mod.</param>
+        public GameID( UInt32 nAppID, string modPath )
+            : this(0)
+        {
+            AppID = nAppID;
+            AppType = GameType.GameMod;
+            ModID = BitConverter.ToUInt32(CryptoHelper.CRCHash(System.Text.Encoding.ASCII.GetBytes(modPath)), 0);

```

Comment:
> `BitConverter.ToUInt32(CryptoHelper.CRCHash`

So we calculate an int, convert it to bytes, and then convert it back to an int?